plugins/protocolbuffers/cpp/v21.7/buf.plugin.yaml

```diff
@@ -6,3 +6,4 @@ description: Base types for C++. Generates message and enum types.
 output_languages:
   - cpp
 spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v21.7/LICENSE
```

Comment:
When I looked earlier, this is the only plugin with a custom license (BSD-3-Clause plus an added statement at the end regarding generated code.